ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -831,26 +832,45 @@ export function D3Blueprint(container, options) {
 
         let relationArcs = _relationArcs.selectAll('.relation').data(arcsData, (d) => getPathId(d));
 
+        // Returns node coordinates with adjustment for member spec rotation if present.
+        const getNodeCoordinates = (entity) => {
+            let _node = nodeForEntity(entity);
+            let _x = _node.x;
+            let _y = _node.y;
+
+            if (entity.isMemberSpec()) {
+                _y += _configHolder.nodes.memberspec.circle.cy;
+                if (entity.parent.hasChildren()) {
+                    // follow the rotation as for 'specNodeData'
+                    let _radius = _configHolder.nodes.memberspec.circle.cy;
+                    let _theta = Math.PI * 2 * _configHolder.nodes.memberspec.rotationAngle / 360;
+                    _x += _radius * Math.cos(_theta);
+                    _y -= _radius - Math.abs(_radius * Math.sin(_theta));
```

Comment:
This is an adjustment required to move arcs along with member spec node when it is rotated.